client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-dialog.jsx

```diff
@@ -1,83 +1,95 @@
 /**
  * External dependencies
  */
-var page = require( 'page' ),
-	React = require( 'react' );
+import page from 'page';
+import React, { Component } from 'react';
+import { translate } from 'i18n-calypso';
```

Comment:
That would be awesome, but `localize` creates a new `React.Class` and does not preserve refs. So if we use `localize`, we'll not be able to use the dialog ref in `DisconnectJetpackButton`. 

We can handle that when remove the ref in a future PR.